Mage/src/main/java/mage/util/CardUtil.java

```diff
@@ -884,8 +884,63 @@ public static String getBoostCountAsStr(int power, int toughness) {
         return getBoostCountAsStr(StaticValue.get(power), StaticValue.get(toughness));
     }
 
-    public static String getBoostText(DynamicValue power, DynamicValue toughness, Duration duration) {
-        String boostCount = getBoostCountAsStr(power, toughness);
+    public static String getBoostCountAsStr_phrasing(DynamicValue power, DynamicValue toughness, boolean useX) {
```

Comment:
nit: underscores in method names are not standard. Not much point in "AsStr" when it returns a String either.

can you add doc comments on appropriate usage?